src/main/kotlin/com/example/delivery/controller/UserController.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.delivery.controller
+
+import com.example.delivery.dto.request.CreateUserDTO
+import com.example.delivery.dto.response.UserDTO
+import com.example.delivery.service.UserService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/users")
```

Comment:
```suggestion
@RequestMapping("/users")
```

we can set the global context in config instead of specifying the prefix in each controller